src/rebar_prv_cover.erl

```diff
@@ -343,7 +343,6 @@ cover_compile(State, Dirs) ->
                 ?WARN("Directory ~p error ~p", [Dir, Reason])
         end
     end, Dirs),
-    rebar_utils:cleanup_code_path(rebar_state:code_paths(State, default)),
```

Comment:
Dropped this one because we no longer remove loaded deps, we only set preferential paths. And since cover compilation is done with deps and not plugins, there's no need to put plugins back at the head of the path anymore